### PR TITLE
HOTT-1899 RoO tab modifications when no country is selected

### PR DESCRIPTION
**Pull request by madviv**
- [x] Changes to RoO tab when no country is selected
- [x] UK
- [x] XI

### CODE REVIEW

File: cypress/e2e/RulesOfOrigin/commodityPageRoO.spec.js [hunk 1]
```diff
@@ -1,145 +1,135 @@
+/* eslint-disable no-unused-vars */
 /* eslint-disable new-cap */
 /* eslint-disable max-len */
 describe('| commodityPageRoO - Rules of Origin - copy and links ', {tags: ['config', 'roo-tag']}, function() {
   // RoO tab exists and links work
-  const countries = [''];
+  const countries = ['', 'xi'];
   const service = ['UK', 'EU'];
```
Comment (alexdesi) [inline cypress/e2e/RulesOfOrigin/commodityPageRoO.spec.js:7]: I'd expect `const services` to be ['uk', 'xi']
and 
countries to contain some country codes

File: cypress/e2e/RulesOfOrigin/commodityPageRoO.spec.js [hunk 2]
```diff
@@ -1,145 +1,135 @@
...
   const message = ['exist', 'not.exist'];
   for (let i=0; i<countries.length; i++) {
```
Comment (alexdesi) [inline cypress/e2e/RulesOfOrigin/commodityPageRoO.spec.js:9]: another, more readable way, to loop through countries is:

```
for (const country of countries) {
...
      cy.visit(`${countries}/commodities/0702000007`);
...
}
```

Comment (alexdesi) [reply cypress/e2e/RulesOfOrigin/commodityPageRoO.spec.js:9]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

**Review by alexdesi — APPROVED** [review]
✅ it looks good, I've suggested a couple of potential improvements.

File: cypress/e2e/RulesOfOrigin/commodityPageRoO.spec.js (hunk 1 above)
Comment (madviv) [reply cypress/e2e/RulesOfOrigin/commodityPageRoO.spec.js]: Resolved in second PR

**Pull request merged**